units/sshd.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=OpenSSH server daemon
+Documentation=man:sshd(8) man:sshd_config(5)
```

Comment:
We can probably delete this documentation line since we don't have `man` or man pages in the container.